apps/docs.blocksense.network/components/common/Command.tsx

```diff
@@ -0,0 +1,158 @@
+import { ReactNode, HTMLAttributes, ComponentProps } from 'react';
+
+import { Command as CommandPrimitive } from 'cmdk';
+
+import { cn } from '@/lib/utils';
+import { Icon } from '@blocksense/ui/Icon';
+import { Dialog, DialogContent } from '@/components/common/Dialog';
+
+export const Command = ({
+  className,
+  ...props
+}: ComponentProps<typeof CommandPrimitive>) => (
+  <CommandPrimitive
+    className={cn(
+      'command__container flex h-full w-full flex-col overflow-hidden rounded-md bg-popover text-popover-foreground',
```

Comment:
```suggestion
      'command__container flex h-full w-full flex-col overflow-hidden rounded-md',
```

Can you remove these colors because they are invalid and check the components from below for more invalid colors